qiskit/visualization/matplotlib.py

```diff
@@ -1247,10 +1254,18 @@ def _draw_ops(self, verbose=False):
                             v_ind += 1
                     clbit_b = sorted(xy_plot, key=lambda xy: xy[1])[0]
                     xpos, ypos = clbit_b
+                    if cond_is_bit and self._cregbundle:
+                        cond_reg = self._bit_locations[op.condition[0]]["register"]
+                        ctrl_bit = self._bit_locations[op.condition[0]]["index"]
+                        label = "%s_%s=%s" % (cond_reg.name, ctrl_bit, hex(val))
```

Comment:
Does it make sense to do this in math notation `"$%s_%s$=%s"` so that is comes out a subscript instead of an underscore?

Reply:
Subscript is a good idea. But I just feel that the subscript would be very small. It would look something like this

![image](https://user-images.githubusercontent.com/51048173/131838311-2ae4cab3-f43b-4195-8413-0f0b4d8cc6f4.png)

If this is still fine, I think subscript is more preferable since it also saves some space.

Reply:
I think better to leave it with the underscore so it is more accessible for users with limited eyesight, devices without zoom capabilities, screenshots etc.